pkg/storegateway/series_refs_test.go

```diff
@@ -1116,17 +1116,19 @@ func TestLoadingSeriesChunkRefsSetIterator(t *testing.T) {
 		assert.NoError(t, appender.Commit())
 	})
 
-	testCases := map[string]struct {
-		blockFactory func() *bucketBlock // if nil, defaultTestBlockFactory is used
-		shard        *sharding.ShardSelector
-		matchers     []*labels.Matcher
-		seriesHasher seriesHasher
-		skipChunks   bool
-		minT, maxT   int64
-		batchSize    int
+	type testCase struct {
+		blockFactory                func() *bucketBlock // if nil, defaultTestBlockFactory is used
+		shard                       *sharding.ShardSelector
+		matchers                    []*labels.Matcher
+		seriesHasher                seriesHasher
+		skipChunks, streamingSeries bool
```

Comment:
While making this change I realised that we don't need to test streaming and non streaming both when fetching chunk refs since there is no practical case here for fetching chunk refs with streaming.

Reply:
aren't refs fetched before fetching chunks?

Reply:
Yes. But this test is only about chunk refs, whose logic is the same with and without streaming. We don't have a streaming flag now, just noChunkRefs combined with entire block or overlapping mint/maxt.